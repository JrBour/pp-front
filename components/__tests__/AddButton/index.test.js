import { shallowMount } from '@vue/test-utils'
import AddButton from '@/components/AddButton'

describe('BaseInput', () => {
  let wrapper
  const mockOnChange = jest.fn();
  beforeEach(() => {
    wrapper = shallowMount(AddButton, {
      propsData: {
        text: 'Submit',
      },
    })
  })

  describe('methods', () => {
    it('should emit handle-click method', async () => {
      const button = wrapper.find('button')
      await button.trigger('click');
      expect(wrapper.emitted('handle-click')).toBeTruthy();
   })
  })

  describe('snapshot', () => {
    it('renders properly', () => {
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
