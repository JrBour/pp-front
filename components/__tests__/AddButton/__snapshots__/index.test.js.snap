// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseInput snapshot renders properly 1`] = `
<button>
  <p>Submit</p>
</button>
`;
